packages/map-template/src/components/VisibilitySwitch/VisibilitySwitch.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { useRecoilState, useRecoilValue } from "recoil";
+import primaryColorState from "../../atoms/primaryColorState";
+import './VisibilitySwitch.scss';
+import { ReactComponent as Light2D } from '../../assets/2d-light.svg';
+import { ReactComponent as Dark2D } from '../../assets/2d-dark.svg';
+import { ReactComponent as Light3D } from '../../assets/3d-light.svg';
+import { ReactComponent as Dark3D } from '../../assets/3d-dark.svg';
+import is3DToggledState from "../../atoms/is3DToggledState";
+
+/**
+ * Component responsible for switching between the 2D and 3D features.
+ */
+function VisibilitySwitch() {
+    const primaryColor = useRecoilValue(primaryColorState);
+
+    const [is3DToggled, setIs3DToggled] = useRecoilState(is3DToggledState);
+
+    return (
+        <div className="switch">
+            <div className="switch__button"
```

Comment:
I think we should use `<button>` elements for buttons 😄

Reply:
Hmmmm, it's not really a button in its own definition. More like a switch element. 
What do you think? Should the class be changed or the tag?

Reply:
A switch contains clickable elements. Those can be anchor or button elements.
The floor selector can also be thought of as a switch conceptually, and those are implemented as `<button>` elements 😛 

Having them as actual buttons also improves accessibility, since you will be able to trigger them with the keyboard.

Reply:
Sounds fair, I replaced the `div` tags with `button` tags.